tools/generate-metadata-for-modifyrepo.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# This script is used when building a local repository of RPMs.
+# For modular packages, metadata for them is required.
+# Generate the YAML required when registering that metadata with `modifyrepo_c` command.
+
+set -eu
```

Comment:
```suggestion
set -eux
```

I suggest we add `-x` because of easy to debug.

Reply:
stdoutを利用するツール（stdoutをリダイレクトしてYAMLを作る）で、debugプリントがあると紛らわしいので `x` はつけませんでした！
（debugプリント自体はstderrなのでツールの利用に影響はないのですが。）
debugしたいときは `bash -x tools/generate-metadata-for-modifyrepo.sh` みたいな感じでやる想定です。